imager/imager_images/tests.py

```diff
@@ -0,0 +1,265 @@
+        self.assertEquals(response.status_code, 200)
+        self.assertIn('Stream View', response.content)
+
+        # Verify user1's photos are on the page
+        self.assertIn(self.photo1.picture.url, response.content)
+        self.assertIn(self.photo2.picture.url, response.content)
+
+    def test_other_cant_see_stream(self):
+        # Login
+        self.client.login(username='user2', password='shall_not')
+
+        # Go to stream page as a logged in user
+        response = self.client.get(
+            reverse('stream', kwargs={'pk': self.user.profile.pk}))
+        self.assertEquals(response.status_code, 200)
+
+        # Verify only user1's public photos are on the page
+        self.assertIn(self.photo1.picture.url, response.content)
+        self.assertNotIn(self.photo2.picture.url, response.content)
+
+
+class EditingandUploadingPhotosTestCase(TestCase):
+    def setUp(self):
+        # Setup a couple users with some content
+        # PhotoFactory.reset_sequence()
+        # UserFactory.reset_sequence()
+        # AlbumFactory.reset_sequence()
+
+        self.user = User(username='user1')
+        self.user.set_password('pass')
+        self.user.save()
+
+        self.another_user = User(username='user2')
+        self.another_user.set_password('shall_not')
+        self.another_user.save()
+
+        self.photo1 = PhotoFactory(user=self.user, published='PU')
+        self.photo1.picture = 'something'
+        self.photo1.save()
+        self.photo2 = PhotoFactory(user=self.user, published='PR')
+        self.photo2.picture = 'else'
+        self.photo2.save()
+
+        self.client = Client()
+
+    def test_user_can_upload_photo(self):
+        self.client.login(username='user1', password='pass')
+        response = self.client.get(
+            reverse('upload_photo'))
+        # make sure at upload photo form
+        self.assertEquals(response.status_code, 200)
+        self.assertIn('Upload a photo', response.content)
+        with open(BASE_DIR + "/Frankensteins_monster_Boris_Karloff.jpg", 'rb') as file1:
+            pic_file = SimpleUploadedFile(file1.name, file1.read(), 'image/jpeg')
+            response = self.client.post(
+                reverse('upload_photo'),
+                {'title': 'My Pets',
+                 'picture': 'String',
+                 'user': self.user,
+                 'published': 'PU'})
+        self.assertIn('Library', response.content)
+        # pic_file = SimpleUploadedFile("pictures.jpg", bin(2413241))
+        # form = response.context['form']
+        # form.fields['title'] = 'Title1'
+        # form.fields['picture'] = pic_file
+        # form.fields['user'] = self.user
+        
+        # response = self.client.post(reverse('upload_photo'), form)
+
+
+        # self.assertFieldOutput(EmailField, {'a@a.com': 'a@a.com'}
+        # self.assertFormError(response, form, 'title', errors)
+        #print Photo.objects.all()
```

Comment:
please don't leave commented code in your files.  It looks sloppy and incomplete.